draft-ietf-jose-hpke-encrypt.md

```diff
@@ -230,10 +229,13 @@ Where:
 
  - `×||y` is the concatenation of byte strings x and y.
  - len32(x) is number of bytes in x as four-byte big-endian integer.
- - enc is the value of "enc" header parameter in JOSE header.
+ - enc is the value of "enc" header parameter in JOSE header. The integrity-protected 'enc' parameter provides protection against an
+   attacker who manipulates the encryption algorithm in the 'enc' parameter.
  - apu is The value of "apu" header parameter if present in JOSE header, otherwise empty string.
  - apv is The value of "apv" header parameter if present in JOSE header, otherwise empty string.
 
+ TBD: Authenticated key agreement mechanisms, such as ECDH-SS or authenticated HPKE modes, mitigate the risk of misusing apu and apv by binding the derived key to the specific identities of the participants. This ensures that any alteration to apu or apv invalidates the derived key, preventing unintended use. However, in the base mode of HPKE, where no authentication is provided, the use of apu and apv does not offer any security guarantees and could be subject to misuse. Do we really need apu and apv in the Recipient Context and why is it not required for JWE Integrated Encryption ?
```

Comment:
```suggestion

```

File issues in GitHub - not as text in the draft.

Reply:
Removed text.